modules/document_repository/templates/menu_document_repository.tpl

```diff
@@ -316,9 +316,13 @@
                             <label class="col-xs-4" for="site">Site<font color="red"><sup> *</sup></font></label>
                             <div class="col-xs-8">
                                 <select name="site" id = "site" class = "form-fields form-control input-sm">
-                                <option value=" "> </option>
+                                <option value=""> </option>
                                     {foreach from = $Sites item=val key=k}
-                                        <option value={$k}>{$val}</option>
+                                        {if $val == "Any"}
+                                            <option value="0">Any</option>
```

Comment:
@johnsaigle 
PHP Fatal error:  Uncaught DatabaseException: Insert statement did not execute successfully: Array\n(\n    [0] => HY000\n    [1] => 1366\n    [2] => Incorrect integer value: ' ' for column 'parent_id' at row 1\n)\n in /var/www/lorisinstall/Loris/php/libraries/Database.class.inc:314\nStack trace:\n#0 /var/www/lorisinstall/Loris/php/libraries/Database.class.inc(243): Database->_realinsert('document_reposi...', Array, true)\n#1 /var/www/lorisinstall/Loris/modules/document_repository/ajax/addCategory.php(60): Database->insert('document_reposi...', Array)\n#2 /var/www/lorisinstall/Loris/htdocs/AjaxHelper.php(93): require('/var/www/lorisi...')\n#3 {main}\n  thrown in /var/www/lorisinstall/Loris/php/libraries/Database.class.inc on line 314, referer: